test/optim/test_lrscheduler.py

```diff
@@ -2285,6 +2286,41 @@ def test_lr_scheduler_verbose_deprecation_warning(self, LRClass):
             LRClass(self.opt)
 
 
+    @parametrize("LRClass", [
```

Comment:
Can you make a one off test to make sure you can load an old state dict that is hardcoded? Just to make sure the BC handling is working as expected?